numpy/__init__.pyi

```diff
@@ -878,9 +878,9 @@ _ByteOrderChar: TypeAlias = L["<", ">", "=", "|"]
 # can be anything, is case-insensitive, and only the first character matters
 _ByteOrder: TypeAlias = L[
     "S",                # swap the current order (default)
-    "<", "L", "little", # little-endian
+    "<", "L", "little",  # little-endian
```

Comment:
Hmm, this messes up the alignment. OTOH, we don't usually align these things.

Reply:
` _DTypeNum` comments are not aligned either.